nixos/doc/manual/release-notes/rl-1709.xml

```diff
@@ -78,6 +78,14 @@ rmdir /var/lib/ipfs/.ipfs
 </programlisting>
     </para>
   </listitem>
+  <listitem>
+    <para>
+      The <literal>postgres</literal> superuser name has changed from <literal>root</literal> to <literal>postgres</literal> to more closely follow what other Linux distributions are doing.
+    </para>
+    <para>
+      The <literal>postgres</literal> default <literal>dataDir</literal> has changed from <literal>/var/lib/postgres</literal> to <literal>/var/db/postgresql/$psqlSchema</literal> where $psqlSchema is 9.6 for example.
```

Comment:
The rename described here is the wrong way around (my fault). It should be `from /var/db/postgres to /var/lib/postgres/$psqlSchema`